### ISSUE TITLE
Docker Support

### ISSUE THREAD

**Issue by donjuanmon** [timebertt/pi-cloud-init#7]
Any plans to support a docker vagrant environment? The virtualbox dependency breaks for anyone using an M1 architecture.

**Comment by timebertt** [comment]
No. Haven't tried it out so far. Feel free to contribute, once you got something working :)
